### DISCUSSION TITLE
Solução   // Stock Control Solution

### DISCUSSION THREAD

**Discussion by LesterCerioli** [LesterCerioli/StockControl#2] [Ideas]
## Assinatura da Solução

A solução de Controle de estoque será um "SAS -- Software as Service e para tal
será necessário que o usuário tenha uma assinatura deste serviço.

Não necessariamente a solulçao será paga ou vendida mas seu uso e consumo estará atrelado à um cadastro de usuário.

Entende-se como assinatura de usuário como aquele responsável pelo uso da solução por parte de uma empresa ou organização.

## Solution Signature

The Inventory Control solution will be a "SAS -- Software as Service and for that
it will be necessary for the user to have a subscription to this service.

Not necessarily the solution will be paid or sold but its use and consumption will be linked to a user registration.

A user signature is understood as the person responsible for the use of the solution by a company or organization.

**Comment by MadalenaDiasSec** [comment]
Eu entendo que podemos construir um microserviço denominado Assinaturas para que
tenhamos um contexto limitado com sua base de dados afim de prover independência
de operação assim como alta-disponibilidade.

Precisamos prover a funcionalidade de que o usuário possa criar os grupos de usuários
atribuindo funções que serão os limites de acesso e operação na plataforma.



I understand that we can build a microservice called Subscriptions so that
we've a limited context with your database in order to provide independence
operation as well as high availability.

We need to provide the functionality that the user can create user groups
assigning functions that will be the limits of access and operation on the platform.
